titus-ext/elasticsearch/src/main/java/com/netflix/titus/ext/elasticsearch/ElasticsearchTaskDocumentPublisher.java

```diff
@@ -144,7 +145,7 @@ public void enterActiveMode() {
     }
 
     private void publishTaskDocuments(List<TaskDocument> taskDocuments) {
-        lastPublishedTimestamp.set(registry.clock().wallTime());
+        timeSinceLastPublished.set(registry.clock().wallTime());
```

Comment:
`lastPublishedTimestamp` was correct as this variables represents the timestamp of the last published action.